FirebaseRemoteConfig/Tests/SwiftAPI/RemoteConfigConsole.swift

```diff
@@ -0,0 +1,244 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+class RemoteConfigConsole {
+  private let projectID: String
+  private var latestConfig: [String: Any]!
+
+  public var requestTimeout: TimeInterval = 10
+
+  private var consoleURL: URL {
+    let api = "https://firebaseremoteconfig.googleapis.com"
+    let endpoint = "/v1/projects/\(projectID)/remoteConfig"
+    return URL(string: api + endpoint)!
+  }
+
+  // TODO: Cleanup once the GHA workflow works
```

Comment:
TODO?

Reply:
I went back and made that initializer a bit cleaner. See [here](https://github.com/firebase/firebase-ios-sdk/blob/8a687101f6c7d68421a63250694fc8b3ae994e86/FirebaseRemoteConfig/Tests/SwiftAPI/RemoteConfigConsole.swift#L27).